docs/Installation/get_started.md

```diff
@@ -0,0 +1,463 @@
+
+So there are basically two steps:
+
+ * Create the flist
+ * Start the container
+
+## 6.1 Creating the flist
+
+Creating a flist is done in two steps:
+
+* First we need to assemble the files and folders we need in the resulting filesystem into a tar.gz archive locally
+* Then we need to publish that archive to the hub (https://hub.grid.tf)
+The second step will actually result in the creation the flist.
+
+As we already have a docker image we will take advantage of our [docker to flist](https://hub.grid.tf/docker-convert) application. It will automatically create the tar.gz archive and publish it on the hub for us !
+
+Go to the [docker converter page](https://hub.grid.tf/docker-convert) and login (you will need to register first).
+
+Then enter the docker image `nginxdemos/hello` in the corresponding input field and click the __convert docker image__ button.
+You should see a success message with your flist url like this
+
+```
+Well done
+Your flist nginxdemos-hello-latest.flist is available.
+
+To start a container, use the following informations:
+
+Source: https://hub.grid.tf/ben_mat_1/nginxdemos-hello-latest.flist
+Storage: zdb://hub.grid.tf:9900
+```
+
+## 6.2 Start the container
+
+We will create a container on our previsously reserved zos virtual machine.
+If you can connect to your zos vm (see above) you can start making a container with our newly created flist.
+
+```python
+JSX> j.clients.zos.my_vm.containers.create(name="simple-nginx-website", flist="https://hub.grid.tf/ben_mat_1/nginxdemos-hello-latest.flist", ports={'8081':80})
+JSX> site = vm.containers.get(name="simple-nginx-website")
+JSX> site.is_running()
+True
+```
+Note that we have mapped the port 80 on the container to the port 8081 on the host vm _ports={host:container}_
+
+That's it ! If everything works as expected, you should be able to reach your nginx served website via http://[your_zos_ip]:8081
+
+### Stream logs
+First we need to find the id of the nginx process running in our container
+let's check all the processes that are running on our container
+```python
+JSX> site.client.job.list()
+[
+    {'cpu': 0, 'rss': 3555328, 'vms': 14069760, 'swap': 0, 'starttime': 1559737984120, 'cmd': {'id': 'entry', 'command': 'core.system', 'arguments': {'args': ['-g', 'daemon off;'
+                ], 'dir': '/', 'env': {'NGINX_VERSION': '1.13.8', 'PATH': '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
+                }, 'name': 'nginx'
+            }, 'queue': '', 'stream': False, 'tags': None
+        }, 'pid': 9
+    },
+]
+```
+Here above we can see that the nginx process has the id _entry_.
+Let's stream the output of the process
+```python
+JSX>sub = site.client.subscribe('entry')
+JSX>sub.stream()
+10.244.211.254 -[1/Jun/2019:13:55:49 ] "GET / HTTP/1.1" 200 7225 "-" "Mozilla/5.0 (X11; Linux x86_64) Chrome/"
+```
+
+# 7. Go further with Jumpscale X
```

Comment:
Maybe we can add a link to the other documentation about capacity reservation ? https://github.com/threefoldfoundation/info_grid/blob/development/docs/capacity_reservation/README.md